src/pretix/presale/templates/pretixpresale/event/resend_link.html

```diff
@@ -0,0 +1,31 @@
+{% extends "pretixpresale/event/base.html" %}
+{% load i18n %}
+{% load bootstrap3 %}
+{% block title %}{% trans "Resend order links" %}{% endblock %}
+{% block content %}
+    <h2>
+        {% trans "Resend order links" %}
+    </h2>
+    <div class="row">
+        <div class="panel-body">
+        {% blocktrans trimmed %}
+            If you lost the link to your order or orders, please enter the email address you
+            used for your order. We will send you an email with links to all orders you placed
+            using this email address.
+        {% endblocktrans %}
+    </div>
+    </div>
+    <div class="row">
+    <form class="form-horizontal" method="post">
+        {% csrf_token %}
+        <div class="col-md-3 col-xs-12">
+            <div class="panel-body">
+                {% bootstrap_form form layout="inline" %}
```

Comment:
The form doesn't yet look very good as it is very small in horizontal size. I'd prefer it to be in one line, e.g. in an analogy to the voucher redemption form.